src/server/command_registry.cc

```diff
@@ -113,22 +115,32 @@ void CommandRegistry::Init(unsigned int thread_count) {
 }
 
 CommandRegistry& CommandRegistry::operator<<(CommandId cmd) {
-  string_view k = cmd.name();
-  auto it = cmd_rename_map_.find(k);
+  auto k = cmd.name();
+
+  absl::InlinedVector<std::string_view, 2> maybe_subcommand = StrSplit(cmd.name(), " ");
```

Comment:
nit: absl can also split into a pair<string,string> 😅

Reply:
that's true but it's more verbose, and I am not sure if `CTAD` will deduce this to `<string_view, string_view> and not <string, string>`